### DISCUSSION TITLE
Typescript and React Native - Cannot find name 'MutationObserver'

### DISCUSSION THREAD

**Discussion by Brendan-csel** [react-hook-form/react-hook-form#2805] [Q&A] [answered]
Greatly appreciated if anyone can offer a suggestion for this issue.

I'm getting `Cannot find name 'MutationObserver'` trying to build a react-native project with react-hook-form 6.7.1.

I assume this relates to the `lib` setting in `.tsconfig`.  Currently the only lib I've got in mine is `esnext`.  

Does anyone know what else I need to include (assuming MutationObserver is implemented on the react-native runtime engines)?

**Comment by bluebill1049** [comment, marked as answer]
looks like we missing the mutation observer https://github.com/react-hook-form/react-hook-form/issues/2419 

https://github.com/react-hook-form/react-hook-form/issues/2419

would you like to send a PR for this?

**Reply by Brendan-csel** [reply]
I had a go at creating a PR https://github.com/react-hook-form/react-hook-form/pull/2807

**Reply by bluebill1049** [reply]
thanks buddy
